frame/identity/src/benchmarking.rs

```diff
@@ -44,7 +44,7 @@ fn add_registrars<T: Config>(r: u32) -> Result<(), &'static str> {
 		let _ = T::Currency::make_free_balance_be(&registrar, BalanceOf::<T>::max_value());
 		let registrar_origin = T::RegistrarOrigin::successful_origin();
 		Identity::<T>::add_registrar(registrar_origin, registrar_lookup)?;
-		Identity::<T>::set_fee(RawOrigin::Signed(registrar.clone()).into(), i, 10u32.into())?;
+		Identity::<T>::set_fee(RawOrigin::Signed(registrar.clone()).into(), i, 0u32.into())?;
```

Comment:
Can we please change this back?

Reply:
There are sometimes optimization when the transferred amount is 0.

Reply:
> Can we please change this back?

Gone ahead to fix this